test/util/util.js

```diff
@@ -2,6 +2,13 @@ import sinon from 'sinon';
 
 export const mockLogger = () => ({ log: sinon.fake(), error: sinon.fake() });
 
+export const mockLocation = (mockedProperties) => new Proxy({}, {
+  get: (_, name) =>
+    (Object.prototype.hasOwnProperty.call(mockedProperties, name)
+      ? mockedProperties[name]
+      : window.location[name])
+});
```

Comment:
For the tests of the CTAs, I want to mock `window.location.reload()`. `window.location` doesn't make that easy for us: `sinon.replace()` isn't enough. That's why I'm using this proxy. It allows us to mock specific properties like `reload` while returning actual values from `window.location` for other properties.